pkg/sshd/server_test.go

```diff
@@ -8,6 +10,7 @@ import (
 	"github.com/deis/builder/pkg/controller"
 
 	"github.com/Masterminds/cookoo"
+	"github.com/arschles/assert"
```

Comment:
Ok, cool with you if I put in an issue to fork it & change the deps? It's a fairly far-reaching change in this repo.

Also, fwiw, we're dependent on lots of packages we don't control ;)